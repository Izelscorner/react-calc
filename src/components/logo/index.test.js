import React from 'react';
import { shallow } from 'enzyme';
import Logo from './index';
import toJson from 'enzyme-to-json';

describe('<Logo />', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Logo url={"test"} />);
  });

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders the logo and src', () => {
    const img = wrapper.find('img');
    expect(img).toHaveLength(1);
    expect(img.prop('src')).toEqual('test');
  });

});
