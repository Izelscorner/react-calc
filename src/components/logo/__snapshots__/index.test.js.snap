// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> matches snapshot 1`] = `
<img
  alt=""
  className="logo"
  src="test"
/>
`;
